pages/patterns/delete.js

```diff
@@ -1,48 +1,93 @@
 import React from 'react';
 import { Heading } from '@twilio-paste/core/heading';
 import { Text } from '@twilio-paste/core/text';
-import { Button } from '@twilio-paste/core/button';
 import { Paragraph } from '@twilio-paste/core/paragraph';
-import { Table, TBody, Th, THead, Tr, Td } from '@twilio-paste/core/table';
 import { useToaster, Toaster } from '@twilio-paste/core/toast';
-import { DeleteIcon } from '@twilio-paste/icons/cjs/DeleteIcon';
+
+import { DeleteTable } from '../../components/site/patterns/DeleteTable';
+import { DeleteConfirm } from '../../components/site/patterns/DeleteConfirm';
 
 export default function Delete() {
+  const severity = 'medium';
+
   const [data, setData] = React.useState([
     {
       id: 1,
-      friendlyName: 'First Service',
-      sid: 4392908903,
+      friendlyName: (
+        <Text as="span" fontFamily="fontFamilyText">
+          Fiiiiiiiiirst thing
+        </Text>
+      ),
+      sid: (
+        <Text as="span" fontFamily="fontFamilyCode">
+          4392908903
+        </Text>
+      ),
     },
     {
       id: 2,
-      friendlyName: 'Second Service',
-      sid: 8439025738,
+      friendlyName: (
+        <Text as="span" fontFamily="fontFamilyText">
+          Second thing
+        </Text>
+      ),
+      sid: (
+        <Text as="span" fontFamily="fontFamilyCode">
+          8439025738
+        </Text>
+      ),
     },
     {
       id: 3,
-      friendlyName: 'Third Service',
-      sid: 8439025703,
+      friendlyName: (
+        <Text as="span" fontFamily="fontFamilyText">
+          Third thing
+        </Text>
+      ),
+      sid: (
+        <Text as="span" fontFamily="fontFamilyCode">
+          8439025738
+        </Text>
+      ),
     },
     {
       id: 4,
-      friendlyName: 'Fourth Service',
-      sid: 6239485739,
+      friendlyName: (
+        <Text as="span" fontFamily="fontFamilyText">
+          last thing
+        </Text>
+      ),
+      sid: (
+        <Text as="span" fontFamily="fontFamilyCode">
+          6239485739
+        </Text>
+      ),
     },
   ]);
 
   const toaster = useToaster();
+  const [isOpen, setIsOpen] = React.useState(false);
+  const [serviceToDelete, setServiceToDelete] = React.useState({});
 
-  const handleDelete = (service) => {
+  const deleteService = (serviceObj) => {
     toaster.push({
-      message: `${service.friendlyName} was successfully deleted.`,
+      message: `${serviceObj.friendlyName.props.children} was successfully deleted.`,
       variant: 'success',
       dismissAfter: 3000,
     });
-    const updatedServices = data.filter((serviceObj) => serviceObj.id !== service.id);
+    const updatedServices = data.filter((service) => service.id !== serviceObj.id);
     setData(updatedServices);
   };
 
+  const handleDelete = (serviceObj) => {
+    if (severity === 'low') {
+      deleteService(serviceObj);
+    } else if (severity === 'medium') {
+      setIsOpen(true);
+      setServiceToDelete(serviceObj);
```

Comment:
I think you might need the opposite of this inside of `deleteService`. From what I can tell you only ever add to `serviceToDelete`... I mgiht be missing it though

Reply:
Oh, I see. It doesn't add, it replaces the service to delete.

Still might be a good idea to clear that piece of state once it's deleted though potentially

Reply:
great idea! we are now setting the serviceToDelete to an empty object after it is successfully deleted from the table